integration/cri-o/cri-o.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/crio_skip_tests.sh"
+
+echo "Obtain CRI-O repository"
+go get -d github.com/kubernetes-incubator/cri-o || true
+
+OLD_IFS=$IFS
+IFS=''
+
+# Skip CRI-O tests that currently are not working
+pushd $GOPATH/src/github.com/kubernetes-incubator/cri-o/test/
+for i in ${skipCRIOTests[@]}
+do
+	sed -i '/'${i}'/a skip \"This is not working\"' "$GOPATH/src/github.com/kubernetes-incubator/cri-o/test/ctr.bats"
```

Comment:
Neat! Is there a general issue number you could add to the skip message out of interest?